src/Metrics/MetricsCalculator.php

```diff
@@ -110,7 +110,7 @@ public function collect(MutantExecutionResult ...$executionResults): void
                     break;
 
                 case DetectionStatus::NOT_COVERED:
-                    $this->notCoveredByTestsCount++;
+                    $this->notExecutedByTestsCount++;
```

Comment:
I'm -1 on this renaming

* it's clear what `covered` means, I don't think there is any confusion in _"coverage notion"_
* now it's consfusing why the property is named `notExecutedByTestsCount`, but the detection status is `NOT_COVERED`

Reply:
>it's clear what covered means,

It's not, because it is _not_ covered. The line code coverage only guarantees that a line is _executed_ by a given test, it does not mean this test covers this line.

Also there was a discussion recently with the other mutation testing tools about an alternative to the kill/escape vocabulary and the resounding candidate was covered/uncovered which is going to be even more confusing if we keep those words with the line coverage

I totally agree with you about the detection status, I would like to change it, but that requires a bit more refactoring affecting the UI as well hence I left it alone for now.

Reply:
> The line code coverage only guarantees that a line is executed by a given test, it does not mean this test covers this line.

I don't agree. I know the difference between types of coverage, and PHPUnit has *line coverage*, which exactly means that line is covered as soon as **any** of the opcode is executed.